app/views/courses/EnrollmentsView.coffee

```diff
@@ -102,6 +93,29 @@ module.exports = class EnrollmentsView extends RootView
     @state.set('totalCourses', @courses.size())
     super()
 
+  onPrepaidsSync: ->
+    @prepaids.each (prepaid) =>
+      prepaid.creator = new User()
+      # We never need this information if the user would be `me`
+      if prepaid.get('creator') isnt me.id
+        @supermodel.trackRequest prepaid.creator.fetchCreatorOfPrepaid(prepaid)
+
+    @decideUpsell()
+
+  onLeadPriorityResponse: ({ priority }) ->
+    @state.set({ leadPriority: priority })
+    @decideUpsell()
+
+  decideUpsell: ->
+    nonStarterLicensePrepaids = @prepaids.filter((p) => p.get('type') != 'starter_license')
```

Comment:
There are non-course prepaids, so you probably want to flip this to be `fullLicenses = @prepaids.filter((p) => p.get('type') is 'course')

Reply:
Ok - I updated the logic to reflect this and added a comment to capture.  Also captured this functionality in tests.  I think this implementation is correct based on my (new) understanding of Prepaid types - if not lets sync offline for a few min I think talking through will be more efficient.